### PR TITLE
Linters project

### PR DESCRIPTION
**Pull request by stanleySimeon**
# In this milestone I

  - ## created a simple HTML project with:
    - a header title
    
  - ## also have:
 
    - a CSS file for the style
    - a README file that contains the documentation of the project
    - a .gitignore file

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (micheaol) [inline test.md]: Please add the test.md file to the .gitignore file.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @mstanley-me 🥇 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### TO HIGHLIGHT: 

- Your project is professional 💯  
- Your repo is professional 💯  
- Well structured files 💯  

## Required Changes ♻️
- _Check the comments under the review._
- Please write a short/meaningful title for your pull request. Please check this [LINK](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for help.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by lily-coder — APPROVED** [review]
Hi @mstanley-me, 

Thank you for implementing changes as requested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (stanleySimeon) [reply test.md]: Okay thanks, I'll do that!

**Pull request merged**